### DISCUSSION TITLE
How to deploy with a custom viz-plugin using helm

### DISCUSSION THREAD

**Discussion by thaarhoff** [apache/superset#27265] [Q&A / Help]
Hello,

in our project, we are using the helm-chart to deploy superset instances.
We now are asked by our customer to create a custom chart. 

The creation of the chart is easy enough. But are not sure if it is possible to use our existing deployment pipeline.

Do you have any suggestions on the correct way to get the new chart into the instance using helm configuration? Preferably without creating a custom docker image :D

Greetings
Thomas

**Comment by viveksingh-numerator** [comment]
hi @thaarhoff , did you got a solution for this requirement?

**Reply by thaarhoff** [reply]
Sadly no. I'd still like a solution without building a custom docker file.
